gr-uhd/lib/usrp_source_impl.h

```diff
@@ -146,6 +146,9 @@ class usrp_source_impl : public usrp_source, public usrp_block_impl
     double _samp_rate;
 
     std::recursive_mutex d_mutex;
+
+    // returns "RX" PMT string for the source
+    const pmt::pmt_t _direction() override;
```

Comment:
You don't have the comment in the usrp_sink -- I think it's also not necessary here.

Reply:
Agree